README.md

```diff
@@ -66,10 +66,11 @@ To switch between modes press `CTRL + /` in search/filter input.
 - `--follow -f` Follow file/stdin. All filters are applied to new data.  
 When navigating up from the end, following will be stopped and resumed on navigating to the end(shift+g) or just by scrolling down till the end
 - `--keep-chars=10`,`-K 10` - Predefines number of kept chars *(see K in key bindings)*
-- `--filters=nginx_php_errors` Specifies path to the file containing predefined filters *(see "Filters" section)*
+- `--filters=nginx_php_errors` Specifies path to the file containing predefined filters or inline filters separated by semicolon *(see "Filters" section)*
```

Comment:
I actually was thinking about "file" or "filters" using same flag.  
But... It actually makes sense this way and I like it
If Go and slit about composition - why not compose this as well :)